.circleci/config.yml

```
@@ -17,30 +17,32 @@ install_official_git_client: &install_official_git_client
   no_output_timeout: "1h"
   command: |
     set -e
-    sudo apt-get update
-    sudo apt-get install -y openssh-client git
+    sudo apt-get -qq update
+    sudo apt-get -qq install openssh-client git
```

If the apt-get fails, does `-qq` suppress the error output?

---

Looks like it's fine.